src/main/java/org/launchcode/liftoffproject/models/DayOfTheWeek.java

```diff
@@ -0,0 +1,11 @@
+package org.launchcode.liftoffproject.models;
+
+public enum DayOfTheWeek {
+    SUNDAY,
+    MONDAY,
+    TUESDAY,
+    WEDNESDAY,
+    THURSDAY,
+    FRIDAY,
+    SATURDAY
+}
```

Comment:
Consider refactoring `DayOfTheWeek.java` enum as follows so that we can have a nicer way to display the days of the week (I think you have some code in the template that tries this but I think this might work better. See https://education.launchcode.org/java-web-development/chapters/enums/enums-in-models.html#add-an-enum-field

```suggestion
public enum DayOfTheWeek {
    SUNDAY("Sunday"),
    MONDAY("Monday"),
    TUESDAY("Tuesday"),
    WEDNESDAY("Wednesday"),
    THURSDAY("Thursday"),
    FRIDAY("Friday"),
    SATURDAY("Saturday")
    
private final String displayName;

EventType(String displayName) {
   this.displayName = displayName;
}

public String getDisplayName() {
   return displayName;
}
}
```